znai-reactjs/src/doc-elements/structure/toc/TableOfContents.js

```diff
@@ -60,7 +64,8 @@ class TableOfContents {
         for (let i = this.tocItems.length - 1; i >= 0; i--) {
             const ti = this.tocItems[i]
             if (ti.fileName === tocItem.fileName && ti.dirName === tocItem.dirName) {
-                return (i - 1) >= 0 ? this.tocItems[i - 1] : null
+                const prev = (i - 1) >= 0 ? this.tocItems[i - 1] : null
+                return prev && this.isIndex(prev) ? null : prev
```

Comment:
I think that `null` and `prev` need to be swapped.

Reply:
logic here and in the test is that "previous page" on a first page will give `null` instead of `index` page

Reply:
Got it.
So if the previous page is empty, return null.
If the previous page is index, return null.
Otherwise return prev.